client/my-sites/upgrades/cart/test/cart-buttons.js

```diff
@@ -1,65 +1,68 @@
 /**
  * External Dependencies
  */
-const sinon = require( 'sinon' ),
-	ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	TestUtils = require( 'react-addons-test-utils' ),
-	{ expect } = require( 'chai' );
+import sinon from 'sinon';
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
```

Comment:
I did, and it wasn't -- I got errors about `preventDefault` acting on undefined, so something was apparently broken with synthetic events...